utils/promis.js

```diff
@@ -24,6 +22,7 @@ const processPromisResults = async (uid) => {
 
     const forms = Object.keys(promisConfig);
     const scoresPayload = {};
+    const scoresPromises = [];
```

Comment:
One promise may not need an array to hold it.

Reply:
there are many promises being generated on the `for` loop starting on line 27

Reply:
There are many promises being generated in the